usr/share/rear/verify/COMMVAULT/default/410_ping_commcell.sh

```diff
@@ -0,0 +1,16 @@
+# ping Commvault server (CommServe)
+
+if test "$PING" ; then
+	read junk CSHOSTNAME junk2 < <(
+		grep CSHOSTNAME /etc/CommVaultRegistry/Galaxy/Instance001/CommServe/.properties
+	)
+	if test "$CSHOSTNAME" ; then
+		ping -c 5 -q "$CSHOSTNAME" >&2
+		StopIfError "Backup server [$CSHOSTNAME] not reachable !"
+	else
+		LogPrint "WARNING ! Could not determine CommServe hostname !"
+		Log "Please check /etc/CommVaultRegistry/Galaxy/Instance001/CommServe/.properties"
```

Comment:
Huh?
A WARNING regardless of
https://schlomo.schapiro.org/2015/04/warning-is-waste-of-my-time.html

Reply:
Not my code, will check if we can convert this to an error :smile:

Reply:
@paspflue I'm worried about hard-coding `Instance001` here, should we not discover that identifier?